src/lib/get-topper-settings.js

```diff
@@ -98,12 +98,18 @@ const usePackageTopper = (content) => {
 	};
 };
 
-const useEditoriallySelectedTopper = (content) => {
+const useEditoriallySelectedTopper = (content, flags) => {
 	let backgroundColour;
+	let myFtButtonVariant = 'standard';
 
 	// Convert old palette colours to new palette colours from Methode
 	if (content.topper.layout === 'full-bleed-offset') {
 		backgroundColour = 'wheat';
+	} else if (content.topper.layout === 'deep-landscape') {
+		if (content.topper.backgroundColour === 'wheat') {
+			backgroundColour = 'white'
+		}
+		myFtButtonVariant = 'opinion'
```

Comment:
Thouth: In this way, you are applying the `opinion` to all the deep-landscape, but I think we want to preserve the original behaviour when the text is white (dark background). Should this part be wrapped by the above conditional `if` statement?

Reply:
The problem is that a lot of articles (for example my test article `5149363c-326d-11e8-ac48-10c6fdc22f03`) have the background color `auto`. Shouldn't the opinion variant automatically select the correct colours depending on background?